inductiva/constants.py

```diff
@@ -9,3 +9,7 @@
 TASK_KILL_MAX_API_REQUESTS = 5
 
 TASK_KILL_RETRY_SLEEP_SEC = 1.0
+
+LOADER_LOAD_PREFIX = "cmd_"
```

Comment:
This is not used.It should be used in `_cli.__init__` 

```suggestion
LOADER_COMMAND_PREFIX = "cmd_"
```